src/main/java/mekanism/client/sound/SoundHandler.java

```diff
@@ -314,6 +314,9 @@ private static class TileTickableSound extends AbstractTickableSoundInstance {
         @Override
         public void tick() {
             // Every configured interval, see if we need to adjust muffling
+            if (Minecraft.getInstance().level == null) {
```

Comment:
This check should be moved below getting a local variable of level, so then you can just do `level == null` instead of the full reference.